params/network_upgrades.go

```diff
@@ -0,0 +1,46 @@
+// (c) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package params
+
+func (c *ChainConfig) forkOrder() []fork {
+	return []fork{
+		{name: "apricotPhase1BlockTimestamp", timestamp: c.ApricotPhase1BlockTimestamp},
+		{name: "apricotPhase2BlockTimestamp", timestamp: c.ApricotPhase2BlockTimestamp},
+		{name: "apricotPhase3BlockTimestamp", timestamp: c.ApricotPhase3BlockTimestamp},
+		{name: "apricotPhase4BlockTimestamp", timestamp: c.ApricotPhase4BlockTimestamp},
+		{name: "apricotPhase5BlockTimestamp", timestamp: c.ApricotPhase5BlockTimestamp},
+		{name: "apricotPhasePre6BlockTimestamp", timestamp: c.ApricotPhasePre6BlockTimestamp},
+		{name: "apricotPhase6BlockTimestamp", timestamp: c.ApricotPhase6BlockTimestamp},
+		{name: "apricotPhasePost6BlockTimestamp", timestamp: c.ApricotPhasePost6BlockTimestamp},
+		{name: "banffBlockTimestamp", timestamp: c.BanffBlockTimestamp},
+		{name: "cortinaBlockTimestamp", timestamp: c.CortinaBlockTimestamp},
+		{name: "durangoBlockTimestamp", timestamp: c.DurangoBlockTimestamp},
+		{name: "cancunTime", timestamp: c.CancunTime}, // XXX: should this be in ethForks?
```

Comment:
see comment, should this be in ethForks? I think this check is missing from subnet-evm

Reply:
yes I think it fits better in ethforks (as we indirectly activate it via `EUpgrade`). In Subnet-EVM there is a `SetEVMUpgrades` that should activate cancun. I put up a draft PR to add Eupgrade and map to Cancun here: https://github.com/ava-labs/subnet-evm/pull/1135/files.